explorer/src/documentation/Main.hs

```diff
@@ -87,7 +87,7 @@ instance ToSchema      C.CBlockEntry
 instance ToSchema      C.CAddressType
 instance ToSchema      C.CAddressSummary
 instance ToSchema      C.CCoin
-instance ToSchema      C.CAda
+instance {-# OVERLAPPING #-} ToSchema      C.CAda
```

Comment:
What happened here? How come these instances were not overlapping before?

Reply:
I don't know. I just saw this error message:

```haskell
/home/fenx/programming/haskell/serokell/cardano-sl/explorer/src/documentation/Main.hs:90:10: error:
    • Overlapping instances for ToSchema (Fixed Data.Fixed.E6)
        arising from a use of ‘Data.Swagger.Internal.Schema.$dmdeclareNamedSchema’
      Matching instances:
        instance Data.Fixed.HasResolution a => ToSchema (Fixed a)
          -- Defined in ‘Data.Swagger.Internal.Schema’
        ...plus one instance involving out-of-scope types
        (use -fprint-potential-instances to see them all)
    • In the expression:
        Data.Swagger.Internal.Schema.$dmdeclareNamedSchema @C.CAda
      In an equation for ‘declareNamedSchema’:
          declareNamedSchema
            = Data.Swagger.Internal.Schema.$dmdeclareNamedSchema @C.CAda
      In the instance declaration for ‘ToSchema C.CAda’
   |        
90 | instance ToSchema      C.CAda
   |          ^^^^^^^^^^^^^^^^^^^^
            
/home/fenx/programming/haskell/serokell/cardano-sl/explorer/src/documentation/Main.hs:99:10: error:
    • Overlapping instances for ToSchema Micro
        arising from a use of ‘Data.Swagger.Internal.Schema.$dmdeclareNamedSchema’
      Matching instances:
        instance Data.Fixed.HasResolution a => ToSchema (Fixed a)
          -- Defined in ‘Data.Swagger.Internal.Schema’
        ...plus one instance involving out-of-scope types
        (use -fprint-potential-instances to see them all)
    • In the expression:
        Data.Swagger.Internal.Schema.$dmdeclareNamedSchema @Micro
      In an equation for ‘declareNamedSchema’:
          declareNamedSchema
            = Data.Swagger.Internal.Schema.$dmdeclareNamedSchema @Micro
      In the instance declaration for ‘ToSchema Micro’
   |        
99 | instance ToSchema      Micro
   |          ^^^^^^^^^^^^^^^^^^^
```

Probably this is because of newer version of `swagger` library.

Reply:
Let's remove our instances then, and check that the behavior is the same?

Reply:
Okay, it's possible to remove some instances and `{-# OVERLAPPING #-}` to make it compilable.